content/Events/community-calendar.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Community Calendar 
+description: Meetings in the Kubernetes Community
+calendar: true
+weight: 1
+---
+{{< calendar >}}
+
+See it on the web at [calendar.google.com](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles) , or paste this [iCal url](https://calendar.google.com/calendar/ical/cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com/public/basic.ics) into any [iCal client](https://en.wikipedia.org/wiki/ICalendar). Do NOT copy the meetings over to a your personal calendar, you will miss meeting updates. Instead use your client's calendaring feature to say you are attending the meeting so that any changes made to meetings will be reflected on your personal calendar. 
```

Comment:
This text should be above the calendar. Folks copying events to their calendar is the # 1 or # 2 problem when it comes to calendar issues. 😬